samples/oauth/index.html

```diff
@@ -59,15 +59,16 @@
       }
 
       forgerock.Config.set({
-        clientId: '<Your OAuth Client>', // string; e.g. 'ForgeRockSDKClient'
-        redirectUri: '<Your Redirect/Callback URL>', // string; e.g. 'https://sdkapp.example.com:8443/_callback'
-        scope: '<Your OAuth Scopes>', // string; e.g. 'openid profile me.read'
+        clientId: 'WebOAuthClient', // e.g. 'ForgeRockSDKClient'
+        redirectUri: 'https://sdkapp.example.com:8443/_callback/', // e.g. 'https://sdkapp.example.com:8443/central-login/'
+        scope: 'openid profile me.read', // e.g. 'openid profile me.read'
         serverConfig: {
-          baseUrl: '<Your AM URL>', // string; e.g. 'https://openam.example.com:9443/openam/'
-          timeout: '<Your Timeout (milliseconds)>' // number; 90000 or less
+          baseUrl: 'https://openam-crbrl-01.forgeblocks.com/am/', // e.g. 'https://openam.example.com:9443/openam/'
```

Comment:
I think is better to sanitise those and not have your env conf in the PR

Reply:
Fixed, thanks for the catch!